grafanalib/core.py

```diff
@@ -213,6 +213,31 @@ def DashboardLink(dashboard, uri, keepTime=True, title=None):
     }
 
 
+def Template(default, dataSource, label, name, query):
```

Comment:
AFAICT is the "default" value for this variable - ie, when you visit the page for the first time.